ovos_gui_plugin_shell_companion/__init__.py

```diff
@@ -190,7 +189,10 @@ def display_advanced_config_groups(self, message=None):
 
     def build_initial_about_page_data(self):
         uname_info = platform.uname()
-        version = get_mycroft_version() or "unknown"
+        try:
+            from ovos_core.version import OVOS_VERSION_STR as version
+        except ImportError:
```

Comment:
TODO: should have a bus event in ovos-core to account for container usage where the import fails